src/utility/StrToArray.ts

```diff
@@ -0,0 +1,8 @@
+export default function str2arr(str: string) {
+	const buf = new ArrayBuffer(str.length * 1); // 2 bytes for each char
+	const bufView = new Uint8Array(buf);
+	for (let i = 0, strLen = str.length; i < strLen; i++) {
+		bufView[i] = str.charCodeAt(i);
+	}
+	return buf;
+}
```

Comment:
move it to src/utility/algod.utility.ts rather than creating its own file, as other function can also be added in it in future

Reply:
oh this file is for making string to array. But we do not need it for now so I will delete it